src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -31,6 +32,14 @@ class PeerActor(nodeInfo: NodeInfo) extends Actor with ActorLogging {
 
   val peerId = self.path.name
 
+  val waitForStatusInterval = 30.seconds
+  val waitForChainCheck = 15.seconds
+
+  //FIXME move this to props
+  lazy val DaoBlockNumber = 1920000
+  lazy val DaoBlockTotalDifficulty = BigInt("39490964433395682584")
```

Comment:
How do you guys feel about putting links to documentation in comments? I for one would like to able to confirm these number and, even more so, would like to read up on the related information.

This also relates to comments in `application.conf` once the numbers get moved there.